ui/src/app/icons/edit/component.spec.ts

```diff
@@ -0,0 +1,64 @@
+import {ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
+import * as ReactTestUtils from 'react-dom/test-utils';
+import {
+  updateAndTick
+} from 'testing/test-helpers';
+import {EditComponent} from './component';
+
+// Note that the description is slightly different from the component for easier test filtering.
+describe('EditIconComponent', () => {
```

Comment:
should these be jest tests and the file be `.spec.tsx`?

Reply:
I don't know - is that part of the conversion process?

Reply:
yes- see (this spec)[https://github.com/all-of-us/workbench/blob/master/ui/src/app/views/quick-tour-modal/component.spec.tsx] for example

Reply:
Done - thank you!